### DISCUSSION TITLE
Flow Control: a programmers text editor

### DISCUSSION THREAD

**Discussion by neurocyte** [dankamongmen/notcurses#2766] [Show and tell]
I wrote a text editor called Flow Control using notcurses, mostly in zig with some bindings I wrote myself. It renders fast and smooth, thanks very much to notcurses. Check it it out here [neurocyte/flow](https://github.com/neurocyte/flow). I think it's a pretty nice showcase of what notcurses can do.

The bindings are not entirely complete and the API perhaps exposes my lacking knowledge of how notcurses is *supposed* to be used, but they work very well for my (13kloc) project so far. Check them out here [neurocyte/notcurses-zig](https://github.com/neurocyte/notcurses-zig). Tested (at least superficially) on linux and macos. This also includes zig packaging to build notcurses itself with zig's integrated C/C++ compiler, so zig projects can just add it as a dependency and produce static binaries with no further build system dependencies. Also works for cross-compiling, although I have only tested this compiling linux-x86_64 -> linux-aarch64 so far.

Thank you for all your hard work!

**Comment by dankamongmen** [comment]
AWESOME! thanks for letting me know, and for using notcurses =] also appreciate the bugs you filed!
